applications/add_file_to_db.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application adds a file to the DB.
+
+    The name and location of the file are required.
+    This application should complement the ones for updating parameters \
+    and adding entries to the DB.
+
+    Command line arguments
+    ----------------------
+    fileName (str or list of str, required)
+        Name of the file to upload including the full path. \
+        A list of files is also allowed, in which case only one -f is necessary, \
+        i.e., python applications/add_file_to_db.py -f file_1.dat file_2.dat file_3.dat \
+        If no path is given, the file is assumed to be in the CWD.
+    directory (str, required if fileName isn't given)
+        A directory with files to upload to the DB. \
+        All files in the directory with a predefined list of extensions will be uploaded.
+    db (str)
+        The DB to insert the files to. \
+        The choices are either the default CTA simulation DB or a sandbox for testing.
+    verbosity (str, optional)
+        Log level to print (default=INFO).
+
+    Example
+    -------
+    uploading a dummy file.
+
+    .. code-block:: console
+
+        python applications/add_file_to_db.py -f data/data-to-upload/test-data.dat
```

Comment:
Does this file exist? I mean, is there a file that we can use to run tests?
If yes, I'd add it to data/test-data

Reply:
No, this was just a documentation example. 

In order to run the test in the unit test I actually create a file on the fly, upload it to the DB, make sure it is there and with the correct ID and then delete it from disk and the DB. See the snippet below

https://github.com/gammasim/gammasim-tools/blob/47061d495df314b9a348dc9709e69e32344d9277/simtools/tests/test_db_handler.py#L285-L296